2024-03-24 17:43:41 INFO [32mpytest tests/frontend/test_e2e_project.py::test_e2e_create_project[firefox] (setup)[0m
curl - X GET 'http://localhost:8111/authenticationTest.html?csrf' 
-H 'User-Agent:python-requests/2.31.0'
-H 'Accept-Encoding:gzip, deflate'
-H 'Accept:*/*'
-H 'Connection:keep-alive'
-H 'Authorization:Basic OjE4NjM1NjE1MjY2MTAwMzczNDE=' 
-d ''
2024-03-24 17:44:17 INFO [32mpytest tests/frontend/test_e2e_project.py::test_e2e_create_project[firefox] (teardown)[0m
curl - X DELETE 'http://localhost:8111/app/rest/projects/id:rhnu61MaDUW' 
-H 'User-Agent:python-requests/2.31.0'
-H 'Accept-Encoding:gzip, deflate'
-H 'Accept:application/json'
-H 'Connection:keep-alive'
-H 'Content-Type:application/json'
-H 'X-TC-CSRF-Token:55bb5a63-f6d6-41dc-8392-4e21cac6b557'
-H 'Cookie:TCSESSIONID=55F2F2B588093B74079C52336C08FE3C'
-H 'Content-Length:0'
-H 'Authorization:Basic OjE4NjM1NjE1MjY2MTAwMzczNDE=' 
-d '' 
 

2024-03-24 17:44:17 INFO 	RESPONSE:
STATUS_CODE: [31m404[0m
DATA: [31mResponding with error, status code: 404 (Not Found).
Details: jetbrains.buildServer.server.rest.errors.NotFoundException: No project found by locator 'count:1,id:rhnu61MaDUW'. Project cannot be found by external id 'rhnu61MaDUW'.
Could not find the entity requested. Check the reference is correct and the user has permissions to access the entity.[0m